temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -525,3 +528,24 @@ message DeleteServiceAccountResponse {
     // The async operation.
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message GetAuditLogsRequest {
+    // The requested size of the page to retrieve - optional.
+    // Cannot exceed 1000. Defaults to 100.
+    int32 page_size = 1;
+    // The page token if this is continuing from another response - optional.
+    string page_token = 2;
+    // Filter for logs inserted at a time >= this.
+    google.protobuf.Timestamp start_time_inclusive = 3;
+    // Filter for logs inserted at a time < this. - optional.
+    google.protobuf.Timestamp end_time_exclusive = 4;
+    // Filter logs by a user's email address - optional.
+    string user_email_substring = 5;
```

Comment:
Are other filters in this file for email address when listing also substring or equality match? Is substring important and/or does it put us in a bind later if we have this in some form of database since substrings are hard to have indexes on?